Dynatrace/dynatrace-operator
# Updates the go linting and pre-commit make targets

# Description
 The go linting make targets were a bit out of date, same for the pre-commit hook. (which now uses the make commands)


## How can this be tested?
Run the make commands
Setup the pre-commit , change stuff, try to commit. (pre commit should not run if no go files were changed)

## Checklist
